### PR TITLE
Fix: Consultation Form Styling Issue and assign a volunteer field in Patient details

### LINKED ISSUE #2850: Consultation Form Styling Issue
the box should not be having a blue border,

### LINKED ISSUE #2862: Styling Issue : Assign to a volunteer in patient detail
remove the blue box from the input box of assign to a volunteer

### PR DESCRIPTION
**Pull request by patelaryan7751**
Fixes #2850 
Fixes #2862 

# Bugs Fixed

## Made the outline green
Screenshot from 2022-06-25 09-10-07

## assign a volunteer field in Patient details made the outline green
image

### CODE REVIEW

**Review by cp-Coder — APPROVED** [review]
LGTM

**Comment by nihal467** [comment]
@patelaryan7751 


remove the border line in the assign to volunteer text box, keep it without any colour for this section

**Comment by nihal467** [comment]
@patelaryan7751 what is the status

**Comment by patelaryan7751** [comment]
Hey, @nihal467 I have done the suggested changes

assign

**Comment by nihal467** [comment]
@patelaryan7751 
image
remove the green border, and keep no colour in the border

**Comment by patelaryan7751** [comment]
Hey, @nihal467 I have done the changes 
consult

**Comment by nihal467** [comment]
test approved

File: src/Components/Common/OnlineUsersSelect.tsx [hunk 1]
```diff
@@ -86,19 +87,28 @@ export const OnlineUsersSelect = (props: Props) => {
...
+                          hidden: !isDropdownExpanded,
+                        }
+                      )}`
+                    : `${clsx(
+                        "py-2 pl-3 w-full focus:outline-none focus:ring-primary-500 focus:border-primary-500",
+                        {
+                          hidden: !isDropdownExpanded,
+                        }
+                      )} `
+                }
```
Comment (khavinshankar) [inline src/Components/Common/OnlineUsersSelect.tsx]:
```suggestion
                className={clsx(
                 "py-2 pl-3 w-full focus:outline-none",
                 outline ? "focus:ring-primary-500 focus:border-primary-500" : "ring-transparent border-transparent",
                 !isDropdownExpanded && "hidden"
                )}
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**